### ISSUE TITLE
DateUtils: Get month implementation too much bloated.

### ISSUE THREAD

**Issue by vijayrawatsan** [jaydeepw/android-utils#6]
Use 
return new SimpleDateFormat("MMMMM").format(date); to get month
Please do relevant null checks as well.

**Comment by jaydeepw** [comment]
Yes. Agreed. Tthats really shitty code written long time back and never looked into again. I was thinking of removing that method altogether and also run static code analysis on it.
